src/main/java/frc/robot/commands/FullAcquireSequenceCommand.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Storage;
+import frc.robot.subsystems.Acquisition;
+
+public class FullAcquireSequenceCommand extends Command {
+    private Storage storage = Storage.getInstance();
+    private Acquisition acquisition = Acquisition.getInstance();
+    private boolean isFinished;
+
+    public FullAcquireSequenceCommand() {
+        addRequirements(storage, acquisition);
+    }
+
+    @Override
+    public void initialize() {
+
+    }
```

Comment:
if you aren't going to put anything in initialize I don't think you should have it